pyproject.toml

```diff
@@ -14,7 +14,7 @@ classifiers = [
 dependencies = [
     'astropy >=5.0.4',
     'scipy >=1.6.0',
-    'numpy >=1.17',
+    'numpy >=1.20',
```

Comment:
I think a dependency change should probably be announced in the changelog (even if its really because other dependencies have pinned  the minimum value above this).